includes/admin/views/html-admin-page-addons.php

```diff
@@ -26,36 +31,82 @@
 	}
 	echo '</ul>';
 }
+?>
+<form class="extension-search" method="get" action="<?php esc_url( admin_url( 'edit.php?post_type=job_listing&page=job-manager-addons' ) ); ?>">
+	<input type="hidden" name="post_type" value="job_listing" />
+	<input type="hidden" name="page" value="job-manager-addons" />
+	<input class="wpjm-extension-search-input" type="text" name="search" value="<?php echo esc_attr( $search ); ?>" placeholder="<?php echo esc_attr__( 'Search', 'wp-job-manager' ); ?>" />
+	<input class="wpjm-extension-search-button button" type="submit" value="<?php echo esc_attr__( 'Search', 'wp-job-manager' ); ?>" />
+</form>
+<?php
 
+echo '</div>';
 echo '<br class="clear" />';
 
 if ( empty( $add_ons ) ) {
 	echo '<div class="notice notice-warning below-h2"><p><strong>' . esc_html__( 'No add-ons were found.', 'wp-job-manager' ) . '</strong></p></div>';
 } else {
 	echo '<ul class="products">';
 	foreach ( $add_ons as $add_on ) {
-		$url = add_query_arg(
-			[
-				'utm_source'   => 'product',
-				'utm_medium'   => 'addonpage',
-				'utm_campaign' => 'wpjmplugin',
-				'utm_content'  => 'listing',
-			],
-			$add_on->link
-		);
+		$class     = '';
+		$link_args = [
+			'utm_source'   => 'product',
+			'utm_medium'   => 'addonpage',
+			'utm_campaign' => 'wpjmplugin',
+			'utm_content'  => 'listing',
+		];
 		?>
 		<li class="product">
-			<a href="<?php echo esc_url( $url, [ 'http', 'https' ] ); ?>">
-				<?php if ( ! empty( $add_on->image ) ) : ?>
-					<img src="<?php echo esc_url( $add_on->image ); ?>" />
+
+			<div class="add-on-header">
+			<?php if ( 'https://wpjobmanager.com/add-ons/bundle/' === $add_on->link ) : ?>
+					<div class="wp-block-group jm-logo__wrapper is-nowrap is-layout-flex">
+						<div class="jm-logo">
+							<svg xmlns="http://www.w3.org/2000/svg" class="jm-logo__head" fill="currentColor" viewBox="0 0 80 80">
```

Comment:
Looks nice, but it's a lot of code for one little image :) Maybe we can put this SVG into a `get_animated_logo()` method to make things more readable.

Reply:
Good idea!  Done in 3fae4d1c710e0efcd90d6440cc77a507b3875550